chevah/github_hooks_server/tests/test_handler.py

```diff
@@ -99,10 +122,13 @@ def test_issue_comment_no_pull(self):
         self.handler.dispatch(event)
 
         self.assertFalse(self.handler.trac.getTicket.called)
+        self.assertLog(
+            '[some] Not a comment on a pull request'
+            )
 
     def test_issue_comment_no_ticket(self):
         """
-        Noting happend is pull request title does not contain a ticket.
+        Noting happens is pull request title does not contain a ticket.
```

Comment:
this docstring is weird.. shouldnt it be **Nothing happens if** ...?

also, the test shows that something happens... there is a log message being triggered, but i'm not sure if it should be part of the specification / docstring